fhirAssembly/src/test/java/io/vantiq/extsrc/fhirAssembly/TestFhirAssembly.java

```diff
@@ -469,9 +508,20 @@ void fetchAndCheckPatient(Vantiq v, String id, String name) throws Exception {
      * ]
      */
     Map<String, ?> findTarget(Vantiq v, String type, Map<String, ?> query) throws Exception {
-        VantiqResponse resp = v.execute("com.vantiq.fhir.fhirService.searchType",
-                                        Map.of("type", type,
-                                               "theQuery", query));
+        VantiqResponse resp;
+        // Alternate calling via GET (the default) vs. POST.  Both methods are allowed, some servers apparently have
```

Comment:
Any way to make this something that gets configured at the service level so it doesn't have to end up on every API call?

Reply:
I'll look at adding such a parameter in a subsequent revision.  Reading the standard docs, it seems to imply that some servers work better with some methods, but may also work better for some particular searches.  Not sure why, but just trying to accommodate what's there.  But adding a "default" override makes sense.

Issue #551

Reply:
That said, in the test it's trying to exercise both forms.  If we allow both, then we'll need to do this -- or run the test suite twice -- once/method.  I'll check with Sam that my understanding of the method stuff is correct.